account_move_line_tax_editable/models/account_move_line.py

```diff
@@ -16,3 +16,24 @@ class AccountMoveLine(models.Model):
     def _compute_is_tax_editable(self):
         for rec in self:
             rec.is_tax_editable = rec.move_id.state == "draft"
+
+    force_tax_line_id = fields.Many2one(
+        "account.tax",
+        string="Originator tax",
```

Comment:
I would set this to "Force Originator Tax" but then in the view use this string to have the UX effect that it is the same field.